Source/WebCore/rendering/RenderLayer.cpp

```diff
@@ -1732,7 +1732,9 @@ bool RenderLayer::updateLayerPosition(OptionSet<UpdateLayerPositionsFlag>* flags
     auto layerRect = computeLayerPositionAndIntegralSize(renderer());
     auto localPoint = layerRect.location();
 
+    bool footprintChanged = false;
```

Comment:
Not a fan of "footprint" (we normally use it in the context of memory). Maybe `geometryChanged` or `sizeOrPositionChanged`.